api.planx.uk/modules/analytics/metabase/shared/client.ts

```diff
@@ -0,0 +1,155 @@
+import axios from "axios";
+import type {
+  AxiosInstance,
+  AxiosError,
+  AxiosRequestConfig,
+  InternalAxiosRequestConfig,
+} from "axios";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+// Custom error class for Metabase-specific errors
```

Comment:
nit: Redundant comment, we can pick this up from a nice and clear class name and the `extend` keyword below 👍